docsource/development.rst

```diff
@@ -114,6 +154,20 @@ It is available only for 12 and 13:
 https://github.com/OCA/OpenUpgrade/pulls?q=is%3Apr+%5BFIX%5D+reset+exception
 
 
+Re-upgrade an upgraded module
+.............................
+
+In some cases, you might want to try forcing the upgrade of a module which was already
```

Comment:
This paragraph is  not related to this PR. Let me know if you want me to create a dedicated PR.

Reply:
Here as well, if you want to try to reset your module versions you have to know what you are doing as it will not work unless with the simpelest of migration scripts.

Reply:
Have you ever been using this trick @StefanRijnhart ? 

This page is for the people who want to do development on OpenUpgrade so somehow they should know what they are doing.

Also, running migration scripts takes quite a lot of time, and I think that this trick can help during development to test faster to be able to move forward. But well, at this stage of learning about OpenUpgrade, what I think is closer to imagination rather than well thought and tested solutions.

While you have way way more experience than me in developing for OpenUpgrade... I probably should just listen to you.

Reply:
I have only ever used this trick when I wanted to iteratively test whether minor migration scripts in custom projects do not raise syntax errors. Such simple scripts are usually reentrant, unlike the complex scripts in OpenUpgrade with their pre/post dependencies and renamed fields.
If this is to ease the development of openupgrade scripts, the trick I use is to raise an error explicitely at the end of the post migration script which rolls back the transaction so that you can safely start again after inspecting the results.